examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6549,6 +6549,10 @@ endpoint 1 {
 
   server cluster Switch {
     emits event SwitchLatched;
+    emits event InitialPress;
```

Comment:
You can't have _any_ of the "*Press" events along with SwitchLatched.  Either this is a latching switch, or a momentary switch.  It can't be both.

Reply:
I think that is why only SwitchLatched event is enabled